custom/icds/views/ccz_hosting.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.utils.translation import (
+    ugettext_lazy,
+)
+from django.utils.functional import cached_property
+from django.utils.decorators import method_decorator
+from django.urls import reverse
+from django.shortcuts import redirect
+
+from corehq import toggles
+from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.locations.permissions import location_safe
+from corehq.apps.domain.decorators import login_and_domain_required
+from custom.icds.forms import (
+    CCZHostingLinkForm,
+)
+from custom.icds.models import CCZHostingLink
+
+
+@location_safe
+@method_decorator([toggles.APP_TRANSLATIONS_WITH_TRANSIFEX.required_decorator()], name='dispatch')
```

Comment:
This was corrected later. I remember i left a comment on the commit about this correction so that the reviewer does get notified too. Not sure if that was not shown.